README.md

```diff
@@ -36,6 +36,8 @@ You can find the following commands in the Command Palette (<kbd>**F1**</kbd> /
 |---|---|---|
 | `java.test.report.position` | Specify where to show the test report. Supported values are: `sideView`, `currentView`. | `sideView` |
 | `java.test.log.level` | Specify the level of the test logs. Supported values are: `error`, `info`, `verbose`. | `info` |
+| `java.test.config` | Specify the configuration for the test cases to run with.<br> More details can be found [here](https://aka.ms/java-test-config). | `{}` |
```

Comment:
Specify the configuration for the test cases to run with.<br>More details can be found [here](https://aka.ms/java-test-config)

vs

Specify the configuration for the test cases to run with. [More details](#)

vs

Specify the configuration for the test cases to run with. <br>[More details](#)

Reply:
Use the second option